src/elements/input/Input.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import styles from "./Input.module.scss";
+
+const Input = ({ type, placeholder, className }) => {
+  return (
+    <input
+      type={type}
+      className={`${styles.input} ${className}`}
+      placeholder={placeholder}
+    ></input>
```

Comment:
use `/>` in the end. No empty body in tags.

Reply:
Noted!